packages/web/src/new/scenes/Main/Auth/Subviews/Create/MnemonicInput.tsx

```diff
@@ -4,9 +4,8 @@ import { useState } from 'react';
 import { styled } from '@linaria/react';
 import { observer } from 'mobx-react-lite';
 
-import { useViewModel } from 'new/core/viewmodels/useViewModel';
+import type { ViewModelProps } from 'new/scenes/Main/Auth/typings';
```

Comment:
We agreed to make relative paths within one logical block. I think here you can change the paths to relative in files from `Main/Auth`